{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -334,6 +335,13 @@
 # django-cors-headers - https://github.com/adamchainz/django-cors-headers#setup
 CORS_URLS_REGEX = r"^/api/.*$"
 
+# By Default swagger ui is available only to admin user. You can change permission classs to change that
+SPECTACULAR_SETTINGS = {
+    "TITLE": "API documentation for{{ cookiecutter.project_slug }}",
+    "DESCRIPTION": "Lets rock and then roll!",
```

Comment:
```suggestion
    "TITLE": "{{ cookiecutter.project_name }} API",
    "DESCRIPTION": "Documentation of API endpoiints of {{ cookiecutter.project_name }}",
```

Reply:
I personally liked the description! :D No worries, I'll change it.